testsrc/org/mozilla/javascript/tests/NativeJavaObjectTest.java

```diff
@@ -0,0 +1,58 @@
+/* -*- Mode: java; tab-width: 8; indent-tabs-mode: nil; c-basic-offset: 4 -*-
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.mozilla.javascript.tests;
+
+import static org.junit.Assert.*;
+
+import java.math.BigInteger;
+import junit.framework.TestCase;
+import org.junit.Test;
+import org.mozilla.javascript.Context;
+import org.mozilla.javascript.NativeJavaObject;
+import org.mozilla.javascript.Scriptable;
+
+public class NativeJavaObjectTest extends TestCase {
+
+    @Test
+    public void testCoerceType() {
+        Context cx = Context.enter();
+        try {
```

Comment:
It's not important to fix now, but for future reference, you can write:

`try (Context cx = Context.enter()) {}`

because Context is Closeable.